apm-agent-plugins/apm-spring-webflux/apm-spring-webflux-spring5/src/test/java/co/elastic/apm/agent/springwebflux/ServletContainerTest.java

```diff
@@ -74,7 +75,7 @@ void shouldOnlyCreateOneTransaction() throws InterruptedException {
         TransactionImpl transaction = reporter.getFirstTransaction(200);
 
         // transaction naming should be set by webflux instrumentation
-        AbstractServerInstrumentationTest.checkTransaction(transaction, "GET /functional/with-parameters/{id}", "GET", 200);
+        AbstractServerInstrumentationTest.checkTransaction(transaction, "GET /functional/with-parameters/{id}", "GET", 200, expectedFrameworkVersion);
```

Comment:
[minor] if it's only used once, then there is no need to define a variable to store it, inline the value for simplicity.

Reply:
I originally had it hardcoded, but made it a variable because of the way the tests inherit from each other - the tests in the spring-webflux-plugin module (that uses Spring 6) extends these test classes 

The tests that use Spring 6 overrides the field in the beforeEach method

Reply:
Ok, then in this case it's fine then.